saksbehandler/src/refusjon/MerkForUnntakOmInntekterFremITid/MerkForUnntakOmInntekterFremITid.tsx

```diff
@@ -59,8 +61,13 @@ const ModalForm: FunctionComponent<{ refusjon: Refusjon; setOpen: (open: boolean
         try {
             await merkForUnntakOmInntekterFremITid(refusjon.id, data.merking);
             setOpen(false);
-        } catch {
-            setError('merking', { type: 'manual', message: 'Noe gikk galt' });
+        } catch (error) {
+            if (error instanceof FeilkodeError) {
+                const feilmeldingTekst = Feilmeldinger[error.message as Feilkode];
+                setError('merking', { type: 'manual', message: feilmeldingTekst });
+            } else {
+                setError('merking', { type: 'manual', message: 'Noe gikk galt' });
+            }
```

Comment:
Vi kunne kanskje utvidet `FeilkodeError`:

```
export class FeilkodeError extends Error {
    feilkode: Feilkode;
    feilmelding: string;

    constructor(feilkode: Feilkode) {
        super(feilkode);
        this.feilkode = feilkode;
        this.feilmelding = Feilmeldinger[feilkode];
    }

    static isFeilkodeError(error: unknown): error is FeilkodeError {
        return error instanceof FeilkodeError;
    }
}
```

Med den endringen så hadde det vært enklere å hente ut meldingen:


```suggestion
        } catch (error) {
            if (FeilkodeError.isFeilkodeError(error)) {
                setError('merking', { type: 'manual', message: error.feilmelding });
            } else {
                setError('merking', { type: 'manual', message: 'Noe gikk galt' });
            }
```